src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -270,7 +270,7 @@ async Task<JObject> ResolveAsLocalOrThisMember(string name)
                     }
                     catch (Exception ex)
                     {
-                        throw new ExpressionEvaluationFailedException($"BUG: Unable to get properties for scope: {scopeId}. {ex}");
+                        throw new ReturnAsErrorException($"BUG: Unable to get properties for scope: {scopeId}. {ex}", ex.GetType().Name);
```

Comment:
Will these now get surfaced in the IDE? How does it show up?

Reply:
This is completely ignored on VS and on chrome, as far as I know we only see the description which was not changed.